### PR TITLE
tpl: Enhance countwords to handle special chars

### PR DESCRIPTION
**Pull request by ResamVi**
Copies the implementation that already is part of .WordCount
This should handle the given test cases from #8479

### CODE REVIEW

File: tpl/strings/strings.go [hunk 1]
```diff
@@ -75,6 +78,11 @@ func (ns *Namespace) CountWords(s interface{}) (int, error) {
 		return 0, _errors.Wrap(err, "Failed to convert content to string")
 	}
 
+	isCJKLanguage := cjkRe.Match([]byte(ss))
```
Comment (bep) [inline tpl/strings/strings.go:81]: Use MatchString (which would save some memory allocation).

Comment (bep) [inline tpl/strings/strings.go:81]: Use MatchString (which would save some memory allocation). Other than that, this looks great.

**Review by bep — CHANGES_REQUESTED** [review]

File: tpl/strings/strings.go (hunk 1 above)
Comment (bep) [reply tpl/strings/strings.go:81]: Also: Could you change the commit message to say "tpl: Fix ..." or something (as this is a bug fix, we use the "fix" word to put them in the right category in the release notes). Also end the commit message with "Fixes #id".

Comment (ResamVi) [reply tpl/strings/strings.go]: Done!

**Pull request merged**